tests/end2end-tests/data_process/movielens_test.sh

```diff
@@ -22,13 +22,13 @@ error_and_exit () {
 
 # Test the DistDGL graph format.
 echo "********* Test the DistDGL graph format ********"
-python3 $GS_HOME/tests/end2end-tests/data_process/process_movielens.py
-python3 -m graphstorm.gconstruct.construct_graph --conf-file $GS_HOME/tests/end2end-tests/data_process/movielens.json --num-processes 1 --output-dir /tmp/movielens --graph-name ml --add-reverse-edges --part-method random
+python3 $GS_HOME/tests/end2end-tests/data_gen/process_movielens.py
+python3 -m graphstorm.gconstruct.construct_graph --conf-file $GS_HOME/tests/end2end-tests/data_gen/movielens_text.json --num-processes 1 --output-dir /tmp/movielens --graph-name ml --add-reverse-edges --part-method random
```

Comment:
Why the output-dir of movielens_text.json is /tmp/movielens but output-dir of data_gen/movielens.json is /tmp/movielens_bert?

Should output-dir of movielens_text.json be /tmp/movielens_with_tokens ?
And output-dir of movielens_text.json be /tmp/movielens_bert_emb?

Reply:
renamed it